### PR TITLE
add "metal user add" command

### LINKED ISSUE #219: metal add user
### What problem are you facing?
I want to add users as admins or collaborators to one or more projects with metal-cli.

### How could the Equinix Metal CLI help solve this problem?
`metal user add foo@example.com` with optional `-p proj_id1 -p proj_id2 -p proj_id3`

Because `metal init` ensures the default project is used, I'd like to get away with using the bare statement above for the majority of use cases.

I’ll need a `--roles` option too (or multiple --role arguments).

### PR DESCRIPTION
**Pull request by displague**
Fixes #219

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,8 @@ go 1.16
 require (
 	github.com/manifoldco/promptui v0.9.0
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/packethost/packngo v0.26.0
+	// TODO: do not merge this. remove this once https://github.com/packethost/packngo/pull/346 is merged
+	github.com/packethost/packngo v0.26.1-0.20220926212055-cc3c8410dbe8 // indirect
```
Comment (displague) [inline go.mod]: Do not merge without addressing this first. Depends on packethost/packngo#346

File: internal/users/add.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			getOpts := c.Servicer.ListOptions([]string{"invited_by", "invitable", "projects"}, nil)
+			createRequest := &packngo.InvitationCreateRequest{
+				Invitee:        email,
+				Roles:          roles,
+				ProjectIDs:     projectIDs,
+				OrganizationID: organizationID,
+			}
+			invitation, _, err := c.InvitationService.Create(createRequest, getOpts)
```
Comment (displague) [inline internal/users/add.go]: I'm having trouble identifying why this API call (with a program id, role, organization id, and invitee) is resulting in:
```
{
 "errors": [
  "Roles contains an invalid role limited_collaborator"
 ]
}
```

Making the same request through the web portal (and investigating the network panel) reveals that the requests are identical.  The differences that I feel need to be further scrutinized for this PR, are the HTTP headers.

Comment (displague) [reply internal/users/add.go]: `project_ids` was supposed to be `projects_ids` 🤦

**Review by ocobles — APPROVED** [review]
lgtm

**Pull request merged**